src/GoTrueClient.ts

```diff
@@ -216,10 +224,10 @@ export default class GoTrueClient {
         token_type,
         user,
       }
-      if (options?.storeSession) {
-        this._saveSession(session)
-        this._notifyAllSubscribers('SIGNED_IN')
-      }
+      if (options?.storeSession) this._saveSession(session)
+      this._notifyAllSubscribers('SIGNED_IN')
+      // Remove tokens from URL
+      window.location.hash = ''
```

Comment:
Since this runs on every page load, won't this have an unintended side-effect of removing legitimate hashes from the URL? For example if you visit 

`https://supabase.io/blog/2020/09/11/supabase-hacktoberfest-2020/#how-do-i-get-started`

And `gotrue-js` is present, it looks like it would strip the title anchor

Reply:
This only runs if the anchor starts with `access_token` (https://github.com/supabase/gotrue-js/blob/feat/expose-api-from-client/src/GoTrueClient.ts#L61) and includes `expires_in`, `refresh_token`, and `token_type` (https://github.com/supabase/gotrue-js/blob/feat/expose-api-from-client/src/GoTrueClient.ts#L212-L215) and even then it only ever runs if the `getUser` API call is successful (https://github.com/supabase/gotrue-js/blob/feat/expose-api-from-client/src/GoTrueClient.ts#L218), so should be pretty safe, no?